### ISSUE TITLE
Unsupported AVI audio format

### ISSUE THREAD

**Issue by Polda18** [gpac/gpac#1387]
Thanks for reporting your issue. Please make sure these boxes are checked before submitting your issue - thank you!

- [x] I looked for a similar issue and couldn't find any.
- [x] I tried with the latest version of GPAC. Installers available at http://gpac.io/downloads/gpac-nightly-builds/
- [x] I give enough information for contributors to reproduce my issue (meaningful title, github labels, platform and compiler, command-line ...). I can share files anonymously with this dropbox: https://www.mediafire.com/filedrop/filedrop_hosted.php?drop=eec9e058a9486fe4e99c33021481d9e1826ca9dbc242a6cfaab0fe95da5e5d95

Detailed guidelines: http://gpac.io/2013/07/16/how-to-file-a-bug-properly/

-------
Hi. Here I am again. I am trying to convert AVI+SRT into MP4 with embedded subtitles, but MP4Box is giving me this error:
```
E:\Videa>mp4box -add "Zpěv na benzínce.avi" "Zpěv na benzínce.mp4"
Unsupported AVI audio format
Error importing Zpěv na benzínce.avi: Feature Not Supported
```
However, according to VLC Media Player, the codec for the video is H.264 and audio is in AAC format, which should be supported:
![Codec informations](https://user-images.githubusercontent.com/25118993/72177511-1cad5080-33e1-11ea-937a-0eb2360dd058.png)

Could it be that the MP4 file doesn't exist? But it should create the file, shouldn't it?

I'm on Windows 10 64bit

**Comment by jeanlf** [comment]
could you upload a sample file?

**Comment by Polda18** [comment]
Okay, I uploaded both of them using FileDrop and clicked Done. Maybe I should have copied the links, I guess? Because now I don't know how to retrieve the files.

**Comment by Polda18** [comment]
Do you have any ideas why it doesn't work? From what I know, the only thing there was issue with is the audio. Video should be fine. However, when I try to add only video (silent), it still gives me this error. And the audio is in supported format, which is AAC. I don't know why it doesn't work. Even trying `-info` yields this error. What's going on?

**Comment by Polda18** [comment]
Ehm. Wasn't the `-v` option supposed to yield more accurate information? Especially for errors?
```
E:\Videa>mp4box -v -info "Zpěv na benzínce.avi"
Unsupported AVI audio format
Error probing file Zpěv na benzínce.avi: Feature Not Supported
```

**Comment by Polda18** [comment]
Okay, I decided to analyze the format online, and it displays this:
```
General
Complete name                            : Zpěv na benzínce.avi
Format                                   : AVI
Format/Info                              : Audio Video Interleave
File size                                : 17.1 MiB
Duration                                 : 5 min 4 s
Overall bit rate                         : 471 kb/s
Writing application                      : Lavf54.29.104

Video
ID                                       : 0
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : Baseline@L3
Format settings                          : 1 Ref Frames
Format settings, CABAC                   : No
Format settings, ReFrames                : 1 frame
Codec ID                                 : avc1
Duration                                 : 5 min 4 s
Bit rate                                 : 355 kb/s
Width                                    : 612 pixels
Height                                   : 344 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 59.940 (60000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.028
Stream size                              : 12.9 MiB (75%)

Audio
ID                                       : 1
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : FF-2
Duration                                 : 5 min 4 s
Bit rate                                 : 96.0 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 44.1 kHz
Frame rate                               : 43.066 FPS (1024 SPF)
Compression mode                         : Lossy
Stream size                              : 3.48 MiB (20%)
Alignment                                : Split accross interleaves
Interleave, duration                     : 23  ms (1.39 video frame)
```
Is it good or bad?

**Comment by jeanlf** [comment]
now fixed in 0.9.0 thanks for the report
